src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -153,6 +155,17 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return chainIdIndices ?? this.implicitChainIdIndices(this.chainId);
   }
 
+  /**
+   * Resolves the lite chain ids that were available to the protocol at a given block range.
+   * @param blockNumber Block number to search for. Defaults to latest block.
+   * @returns List of lite chain IDs that were available to the protocol at the given block number.
+   * @note This dynamic functionality has been added after the launch of Across.
+   */
+  getLiteChainIdIndicesForBlock(blockNumber: number = Number.MAX_SAFE_INTEGER): number[] {
+    const liteChainIdList = sortEventsDescending(this.liteChainIndicesUpdates);
```

Comment:
Should we consider sorting this on update() so that we don't need to do it each time it's queried? I could imagine we might query this a lot in the relayer.

Reply:
Agreed - we also do this in other cases, like for the `disabledChainId` and `chainIdIndices.` It may be worth squeezing more performance by amortizing this in the `update`.

Reply:
➕